ocs_ci/ocs/ui/base_ui.py

```diff
@@ -1485,20 +1499,1143 @@ def __init__(self):
+
+    def open_side_bar_of_entity(self, entity_name: str = None, canvas: bool = False):
+
+        """
+        Opens the side bar of an entity in the topology view.
+
+        Args:
+            entity_name (str, optional): Name of the entity to open the side bar for.
+            canvas (bool, optional): Flag indicating whether to click on the topology graph canvas.
+
+        Note:
+            If `canvas` is True, the method clicks on the topology graph canvas to open the side bar.
+            Otherwise, it searches for the entity and clicks on it. The method attempts to open the
+            side bar up to three times, zooming out the topology view if necessary.
+
+        """
+        if canvas:
+            self.do_click(self.topology_loc["topology_graph"])
+        else:
+            from ocs_ci.ocs.ui.helpers_ui import format_locator
+
+            loc = format_locator(self.topology_loc["select_entity"], entity_name)
+
+            for i in range(1, 4):
+                try:
+                    self.do_click(loc)
+                    break
+                except NoSuchElementException:
+                    logger.info("zooming out topology view")
+                    self.do_click(self.topology_loc["zoom_out"])
+                    self.page_has_loaded(module_loc=self.topology_loc["topology_graph"])
+                    logger.info(f"try read topology again. attempt number {i} ")
+            logger.info(f"Entity {entity_name} sidebar is opened")
+
+    def close_sidebar(self, soft=False):
+        """
+         Closes the sidebar in the topology view.
+
+        Args:
+            soft (bool, optional): If True and the sidebar is visible, closes the sidebar with a transition.
+                                   If False (default) or the sidebar is not visible, closes the sidebar immediately and
+                                   may get exception if sidebar is not open.
+
+        """
+
+        if soft and self.get_elements(self.topology_loc["close_sidebar"]):
+            # sidebar has a slow transition and may be visible for a moment after it was closed
+            try:
+                self.do_click(
+                    self.topology_loc["close_sidebar"], enable_screenshot=True
+                )
+            except TimeoutException:
+                pass
+        elif not soft:
+            self.do_click(self.topology_loc["close_sidebar"], enable_screenshot=True)
+        else:
+            return
+        logger.info("Sidebar is closed")
+
+    def is_node_down_alert_in_alerts_ui(self, entity=None, read_canvas_alerts=False):
+        """
+        Checks if a NodeDown alert is present in the UI alerts.
+
+        Args:
+            entity (str, optional): Entity name to filter alerts (default: None).
+            read_canvas_alerts (bool, optional): Whether to read alerts from the canvas (default: False).
+
+        Returns:
+            (bool): if the node down alert visible in Alerts tab of the Topology
```

Comment:
returns type is not expected to be in brackets
```suggestion
            bool: if the node down alert visible in Alerts tab of the Topology
```

Reply:
noted, I will fix this with next PR, I proceed to develop tests in this area